src/App.js

```diff
@@ -53,15 +53,17 @@ export function App () {
      * If neither are provided, we display a prompt to load a file.
      */
     const init = () => {
-        const urlSearchParams = new URLSearchParams(window.location.search.substring(1));
+        const urlSearchParams = new URLSearchParams(window.location.search);
```

Comment:
It appears that URLSearchParams() can correctly parse a string starting with a "?" but not a "#".

Reply:
Hm, I wonder if it's better to just stay consistent and use substring(1) for both.

Reply:
Sure, since adding .substring(1) is harmless.